register/murmur3/multihash_murmur3.go

```diff
@@ -0,0 +1,23 @@
+/*
+	This package has no purpose except to perform registration of multihashes.
+
+	It is meant to be used as a side-effecting import, e.g.
+
+		import (
+			_ "github.com/multiformats/go-multihash/register/murmur3"
+		)
+
+	This package registers multihashes for murmur3
+*/
+package murmur3
+
+import (
+	"hash"
+
+	multihash "github.com/multiformats/go-multihash/core"
+	"github.com/spaolacci/murmur3"
+)
+
+func init() {
+	multihash.Register(multihash.MURMUR3_128, func() hash.Hash { h := murmur3.New128(); return h })
```

Comment:
If the goal is to make sure we have a description of what unixfsv1 does internally, per [A](https://github.com/multiformats/go-multihash/issues/135#issuecomment-786369575) and [B](https://github.com/ipfs/go-unixfsnode/blob/c7f1fe6ecd168c22f77386a71e8f7eaa647e81fb/hamt/util.go#L108-L112), it appears the function desired is `New64`, not `New128`.

Of course, that's not what multihash indicator code `0x22` is stated to be.  :shrug: